chef/cookbooks/provisioner/templates/suse/crowbar_register.erb

```diff
@@ -122,7 +122,17 @@ add_user() {
 ADMIN_IP="<%= @admin_ip %>"
 ADMIN_BROADCAST="<%= @admin_broadcast %>"
 WEB_PORT="<%= @web_port %>"
-HTTP_SERVER="http://${ADMIN_IP}:${WEB_PORT}"
+<% if @ip_version == "6" -%>
+IPV6=1
+<% else -%>
+IPV6=0
+<% end -%>
+if (( $IPV6 > 0 )); then
+  ADMIN_IP_WRAPPED="[ADMIN_IP]"
+else
+  ADMIN_IP_WRAPPED="ADMIN_IP"
+fi
```

Comment:
Could we collapse these into one if block? They will effectively do the same thing anyway

Reply:
yup, good point. I guess I shouldn't try and write back code that allows someone to easy hack on a machine (turn IPV6 to 0 or 1) cause it's not really making it any better.
DONE